cli/cli.go

```diff
@@ -1,10 +1,13 @@
 package cli
 
 import (
+	"errors"
 	"flag"
 	"fmt"
 	"regexp"
 	"strings"
+
+	"github.com/go-ozzo/ozzo-validation"
```

Comment:
Please add the dependency to the `vendor/` folder.

Reply:
The code below is working for me:

```go
func (c *App) Validate() error {
  if c.Command == "new-alias" {
    return validation.ValidateStruct(c,
      validation.Field(c.Server, validation.Required),
      validation.Field(c.Remote, validation.Required))
  }

  return nil
}
```

Reply:
But it doesn't work for me:
![err](https://i.imgur.com/LYGPmzf.png)

Reply:
Could you check what is the latest commit of `github.com/go-ozzo/ozzo-validation` that you have to your Go workspace? Mine is pointing to `106681dbb37bfa3e7683c4c8129cb7f5925ea3e9` related to tag `v3.5.0`.

Reply:
![img](https://i.imgur.com/2cCLL1L.png)
and here is my `vendor.json`
```
{
  "checksumSHA1": "lZHmhkz+QdNpbGMC05esL0bg3Gs=",
  "path": "github.com/go-ozzo/ozzo-validation",
  "revision": "106681dbb37bfa3e7683c4c8129cb7f5925ea3e9",
  "revisionTime": "2018-09-29T20:53:26Z"
}
```

Reply:
I don't know what `vendor.js` is for, but how about proceed with the implementation  without using  `ozzo-validation`?